envschema/casters.py

```
@@ -20,130 +20,131 @@ def cast_str(value: str) -> str:
 
 def cast_int(value: str) -> int:
     """Кастит значение в целое число.
-    
+
     Args:
         value: Строковое значение из окружения
-        
+
     Returns:
         Целое число
-        
+
     Raises:
         ValueError: Если значение невозможно преобразовать в int
     """
     try:
         return int(value)
     except ValueError:
-        raise ValueError(f"cannot cast '{value}' to int")
+        raise ValueError(f"cannot cast '{value}' to int") from None
```

U sure u want ONLY ur text

---

This follows PEP 409 suppressing exception context when the original exception is not relevant to the caller.
We explicitly use from None to suppress exception chaining and avoid noisy tracebacks from internal casting logic.
The original ValueError from int() does not provide actionable information at this level, so we surface a clean, user-focused error instead

---

Mostly its done when you want to show error to user not engineer
Think about about adding original error